runtime/jailer.go

```diff
@@ -72,5 +76,5 @@ func newJailer(
 	}
 
 	l := logger.WithField("jailer", "runc")
-	return newRuncJailer(ctx, l, ociBundlePath, service.config.JailerConfig.RuncBinaryPath, jailerUID, jailerGID)
+	return newRuncJailer(ctx, l, ociBundlePath, service, jailerUID, jailerGID)
```

Comment:
Not sure if I like passing in the service. We are doing this to get the VMID?

Reply:
We need the config and the vmID, so I figured since we already have a lot of parameters, it made sense to just pass in the service.